schedule/migration/aarch64_regression_test_offline.yaml

```diff
@@ -0,0 +1,109 @@
+name:           aarch64_regression_test_offline.yaml
+description:    >
+  This is for aarch64 offline regression test.
+  #REGRESSION_SERVICE: '1' means support service check test, currently for not
+  # via package media.
+  #REGRESSION_TEST: '1' means regression test part1 include console test. '2'
+  # means regression test part2 include x11 test and some performance related test.
+vars:
+  DESKTOP: 'gnome'
+  BOOT_HDD_IMAGE: 1
+  ORIGIN_SYSTEM_VERSION: '%HDDVERSION%'
+  UPGRADE_TARGET_VERSION: '%VERSION%'
+  BOOTFROM: 'd'
+schedule:
+  - migration/version_switch_origin_system
+  - boot/boot_to_desktop
+  - update/patch_sle
+  - migration/record_disk_info
+  - '{{install_service}}'
+  - migration/reboot_to_upgrade
+  - migration/version_switch_upgrade_target
+  - '{{isosize_test}}'
```

Comment:
Can i ask why this needs to be conditional?

Reply:
This is because for offline migration via package media won't have the module 'isosize' loaded before 'bootloader'.